ghproxy/ghcache/coalesce.go

```diff
@@ -83,15 +82,13 @@ func (coalescer *requestCoalescer) RoundTrip(req *http.Request) (*http.Response,
 	// Only coalesce GET requests
 	if req.Method != http.MethodGet {
 		resp, err := coalescer.requestExecutor.RoundTrip(req)
-		if strings.HasPrefix(req.URL.Path, "graphql") || strings.HasPrefix(req.URL.Path, "/graphql") {
-			var tokenBudgetName string
-			if val := req.Header.Get(TokenBudgetIdentifierHeader); val != "" {
-				tokenBudgetName = val
-			} else {
-				tokenBudgetName = coalescer.hasher.Hash(req)
-			}
-			collectMetrics(ModeNoStore, req, resp, tokenBudgetName)
+		var tokenBudgetName string
+		if val := req.Header.Get(TokenBudgetIdentifierHeader); val != "" {
+			tokenBudgetName = val
+		} else {
+			tokenBudgetName = coalescer.hasher.Hash(req)
 		}
+		collectMetrics(ModeSkip, req, resp, tokenBudgetName)
```

Comment:
can you explain why not using "ModeNoStore" here? Would it be better to distinguish between graphql and other requests?

Reply:
`NO-STORE` is actually a possible `Cache-Control` header value https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cache-Control.
I thought it would be better to properly distinguish a response avoiding the cache due to a `NO-STORE` response header or due to the cache not being applicable for the request type and thus being bypassed.

Reply:
ack